theano/gpuarray/c_code/dnn_fwd.c

```diff
@@ -282,10 +314,16 @@ APPLY_SPECIFIC(conv_fwd)(PyGpuArrayObject *input, PyGpuArrayObject *kerns,
           mathtype = choice.mathType;
 #endif
       } else {
+        #ifdef DEBUG_TIMING
+        timer.start();
```

Comment:
Can you confirm that this reset the timer?

Reply:
Yes, it resets the timer.